src/App.css

```
@@ -1,107 +1,25 @@
 .App {
-  text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background-color: #b9dfed;
+    position:relative;
+}
+
+.Header {
+    position: fixed;
```

```suggestion
    position: sticky;
```
This will also help make sure the modal and filter/search shows.

---

I have never heard of sticky!

---

Yep this definitely helped!